x-pack/plugins/siem/public/components/stat_items/barchart.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  // @ts-ignore
+  EuiSeriesChartUtils,
+} from '@elastic/eui';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+// @ts-ignore
+import { EuiSeriesChart, EuiBarSeries, EuiXAxis, EuiYAxis } from '@elastic/eui/lib/experimental';
+import { BarChartData, WrappedByAutoSizer, ChartHolder } from '.';
+import { AutoSizer } from '../auto_sizer';
+
+const { SCALE, ORIENTATION } = EuiSeriesChartUtils;
+const getYaxis = (value: string | number) => {
+  const label = value.toString();
+  const labelLength = 4;
+  return label.length > labelLength ? `${label.slice(0, labelLength)}.` : label;
+};
+
+export const BarChartBaseComponent = pure<{
+  data: BarChartData[];
+  width: number | null | undefined;
+  height: number | null | undefined;
+}>(({ data, ...chartConfigs }) => {
+  return chartConfigs.width && chartConfigs.height ? (
+    // @ts-ignore
+    <SeriesChart
+      yType={SCALE.ORDINAL}
+      orientation={ORIENTATION.HORIZONTAL}
+      showDefaultAxis={false}
+      data-test-subj="stat-bar-chart"
+      {...chartConfigs}
+    >
+      {data.map(series => {
+        return (
+          // @ts-ignore
```

Comment:
Doesn't look like this `@ts-ignore` is required. I was able to remove it without error.